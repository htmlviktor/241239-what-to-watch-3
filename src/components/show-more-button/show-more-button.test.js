import React from 'react';
import renderer from 'react-test-renderer';
import ShowMoreButton from "./show-more-button";

it(`ShowMoreButton snapshot test`, () => {
  const tree = renderer
    .create(<ShowMoreButton />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
