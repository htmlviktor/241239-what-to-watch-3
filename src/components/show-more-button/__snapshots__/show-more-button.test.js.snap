// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMoreButton snapshot test 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    type="button"
  >
    Show more
  </button>
</div>
`;
